trunk/SUAVE/Methods/Aerodynamics/AVL/run_analysis.py

```diff
@@ -62,6 +62,7 @@ def call_avl(avl_object):
     import sys
     import time
     import subprocess
+    import os
```

Comment:
You have 2 os imports. One in a function and one at the top.

It's bad practice to put imports in a function (with a few exceptions). Please remove the function level imports.

Reply:
Issue resolved